src/math/Vector2.js

```diff
@@ -278,21 +278,14 @@ Object.assign( Vector2.prototype, {
 
 	},
 
-	clampScalar: function () {
+	clampScalar: function (minVal, maxVal) {
```

Comment:
Minor code style issue. It should be:
```js
clampScalar: function ( minVal, maxVa l) {
```

Reply:
In any event, resolving the closure is a good idea in order to reduce the amount of scopes (which can have an impact on performance).

Reply:
OK.